frontend/src/app/app.component.ts

```diff
@@ -1,9 +1,21 @@
-import { Component } from '@angular/core';
+import { DOCUMENT } from '@angular/common';
+import { Component, Inject } from '@angular/core';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
 })
 export class AppComponent {
-  title = 'HTTAT';
+  isDarkMode = false;
```

Comment:
The assumption here is that the system is in light mode when we first come to the site. Could we store the state in localStorage and grab it when someone comes back to the site?

Reply:
this is a much larger task for me that I was trying to avoid 😂.

It was always the plan though. So I'll finish this out before pushing this feature.